shinysdr/test/plugins/wspr/test_demodulator.py

```diff
@@ -86,6 +87,12 @@ def output_message(self, message):
         self.messages.append(message)
 
 
+STATUS_IDLE = 'Waiting for next even minute'
+STATUS_RECEIVING = 'Receiving'
+STATUS_DECODING = 'Decoding…'
+STATUS_DECODING_AND_RECEIVING = STATUS_DECODING
```

Comment:
I introduced these for the sake of not duplicating strings _within the tests_ — I tend to think that tests should actually encode the things they are testing, and in this case, we want to verify _those particular strings_ are displayed to the user. But on further consideration, that is a stronger test than we really need. Moved.